acc-testcases/resources/load_balancer_monitor-monitor_err.yaml

```diff
@@ -0,0 +1,24 @@
+vars:
+  rand_name_1: tf_HTTP_MONITOR_%rand_int
+acc:
+- config: |
+    lb_id = 262 
+    name  =  "$(http_name)"      
+    description  = "HTTP_MONITOR creating using tf"
+    type = "LBHttpMonitorProfile"
+    http_monitor {
+      fall_count = 8
+      interval =  10  
```

Comment:
how is this a negative scenario ? i see the type and the body both are same. what are u trying to check here ?

Reply:
here I am trying to validate for the name value line 2 having variable rand_name_1 for the name value but inside config line 6 variable value is http_name but its not valid one.
now added one sceanrios like the type missmatch for the body.